src/Iris/Cli/Cmd.hs

```diff
@@ -0,0 +1,30 @@
+{- |
+Module                  : Iris.Cli.Cmd
+Copyright               : (c) 2022 Dmitrii Kovanikov
+SPDX-License-Identifier : MPL-2.0
+Maintainer              : Dmitrii Kovanikov <kovanikov@gmail.com>
+Stability               : Experimental
+Portability             : Portable
+
+Wrapper around cmd.
+
+@since x.x.x.x
+-}
+
+module Iris.Cli.Cmd
```

Comment:
Actually, I have a better idea: let's name this module `Iris.Cli.Internal`. It's an internal implementation detail and users don't need to care about what's inside.

There's a convention in the Haskell world that `*.Internal.*` modules are free from following PVP. So we can introduce any sorts of breaking changes in them without the need to worry about backwards compatibility. (and we can remove all the `@since ...` annotations in this module as well.

Reply:
So `Iris.Cli.Internal.Cmd`? And is it the same approach for the other files in the `Iris.Cli.*` module? (`Iris.Cli.Interactive`, `Iris.Cli.ParserInfo`, `Iris.Cli.Browse`, `Iris.Cli.Version`)

Reply:
Just `Iris.Cli.Internal` is enough for now. If we'll have more internal stuff, we can modularise it further 🙂 
For now it's just the `Cmd` type.

Other modules don't need to be `Internal` because they are not internal, users are supposed to use them. And we need to care about setting PVP version properly after changes in those modules

Reply:
Fixed